src/console/CliManager.cpp

```diff
@@ -171,8 +171,8 @@ bool CliManager::readLine(std::string &line, bool linebreak) {
         if (!isInteractive_) {
             break;
         }
-        auto purePrompt = folly::stringPrintf("(%s@%s:%d) [%s]> ",
-                                              username_.c_str(), addr_.c_str(), port_,
+        auto purePrompt = folly::stringPrintf("(%s@) [%s]> ",
```

Comment:
You can replace IP by hostname.

Reply:
Hostname can be resolved by dns

Reply:
I mean the `gethostname` function return value which the '/etc/hostname' content. It's set by system administrator and usually not a domain could be resolved.

Reply:
I mean the `gethostname` function return value which the '/etc/hostname' content. It's set by system administrator and usually not a domain could be resolved to public IP.It could be a good label about machine in the console.

Reply:
1. IP address could be hostname. 2.If hostname can be resolved by dns, the ip will leak